### PR TITLE
Changes wanted by Alan from card

### PR DESCRIPTION
**Pull request by ChuckKollar**
https://github.com/x-atlas-consortia/hs-ontology-api/issues/21#issuecomment-1707149316

### CODE REVIEW

**Review by AlanSimmons — APPROVED** [review]
The changes look good. We will need to regression test against the dev instance of Neo4j, which should still be in the old format for the CodeID property of the Code node (SAB CODE).

File: VERSION
Comment (yuanzhou) [inline VERSION:1]: @ChuckKollar @AlanSimmons I was not requested as a reviewer. However, I should be tagged as one for tracking purposes even if sometimes I may not familiar to the code changes. 

WRT this PR, we should use the version 1.4.1 instead of 1.5.0. We use the [Semantic Versioning 2.0.0](https://semver.org/) as a general guideline. You don't have to bump the version number, consider it optional. I would take care of that by bumping the number in `main` branch then sync into `dev-integrate` for release purposes.

**Review by yuanzhou — CHANGES_REQUESTED** [review]

**Comment by AlanSimmons** [comment]
@yuanzhou Should we set up a CODEOWNERS file in both hs-ontology-api and ubkg-api? As I understand, this results in designated code owners being added automatically as reviewers.

**Comment by yuanzhou** [comment]
> @yuanzhou Should we set up a CODEOWNERS file in both hs-ontology-api and ubkg-api? As I understand, this results in designated code owners being added automatically as reviewers.

Yes CODEOWNERS will have me tagged automatically. We did that in search-api repo since the Harvard and PSC teams contribute to the code changes too. In other repos, the team members just add me to the reviewers list.
If you add that file, please add your github account there too.

**Review by yuanzhou — APPROVED** [review]

**Comment by yuanzhou** [comment]
Note: this PR will go hand in hand with the new set of CSVs. Is this correct @AlanSimmons ?

**Comment by AlanSimmons** [comment]
@yuanzhou The new set of CSVs has a dependency on this PR. The new set of CSVs will not be submitted simultaneously, but after the PR is merged.

The CSVs now represent the CodeID property of Code nodes in format SAB:CODE, instead of the older format of SAB CODE. (The delimiter is now colon instead of space.) Some API endpoints currently assume SAB CODE and will break for a UBKG with the newer format. 

The code in this PR can handle both SAB CODE and SAB:CODE formats. 

Once this PR is merged, I can submit a neo4j in the new format of SAB:CODE.

**Pull request merged**
